src/lib.rs

```diff
@@ -573,6 +591,27 @@ impl Bump {
         self.alloc_with(|| val)
     }
 
+    //TODO: Tests
```

Comment:
I'll try to generalise `try_alloc.rs` a bit, so that the allocator can be shared between different tests.